File: aidltwo/src/main/java/tv/bangumi/aidltwo/activity/MainActivity.java
package tv.bangumi.aidltwo.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.Button;

import tv.bangumi.aidltwo.R;


/**
 * @author 朱峰 2019/8/30
 */
public class MainActivity extends AppCompatActivity {

    private Button bt_one;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.act_main);

        bt_one = (Button) findViewById(R.id.bt_two);
    }
}

File: demo_component_kt/src/main/java/com/billy/cc/demo/component/kt/MainActivity.kt
package com.billy.cc.demo.component.kt

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.Gravity
import android.view.View
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val textView : TextView = TextView(this)
        textView.gravity = Gravity.CENTER
        textView.setText("kotlin page\nclick to finish!")
        textView.setOnClickListener({
            finish()
        })
        setContentView(textView)

        textView.setOnClickListener(object :View.OnClickListener{
            override fun onClick(v: View?) {
                TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
            }
        })
        textView.setOnClickListener{

        }
        textView.setOnEditorActionListener{ v,actionId,event ->

            1==2
        }
    }
}
